skimage/transform/_warps.py

```diff
@@ -65,8 +65,8 @@ def resize(image, output_shape, order=1, mode='reflect', cval=0, clip=True,
         avoid aliasing artifacts.
     anti_aliasing_sigma : {float, tuple of floats}, optional
         Standard deviation for Gaussian filtering to avoid aliasing artifacts.
-        By default, this value is chosen as (1 - s) / 2 where s is the
-        down-scaling factor.
+        By default, this value is chosen as (s - 1) / 2 where s is the
+        down-scaling factor, assuming s > 1.
```

Comment:
otherwise what value does it take?

Reply:
What do you mean ? For the non-default value ?

Reply:
for the default value. so s=1 means no scaling, s>1 means shrinkage, and s<1 means enlargement. What happends when we enlarge photos.

Reply:
As far as I know we're not supposed to use the `anti_aliasing` and `anti_aliasing_sigma` parameters when we enlarge images. So `s` should always be superior or equal to 1 but I might be wrong...

Reply:
Maybe an other sentence like 
> If `s < 1` (up-size), no anti-aliasing is performed prior to rescaling.

Reply:
It is more that you now added a condition `s > 1` to the first sentence. So it begs the question 

> what happens when s < 1???

Reply:
You're right, I'm adding this sentence right now.